### PR TITLE
Use matrix for build, include windows, and unify SHA256 hash with powershell

### PR DESCRIPTION
**Pull request by pearcec**
#243

### CODE REVIEW

File: .github/workflows/cross.yaml [hunk 1]
```diff
@@ -4,63 +4,51 @@ on:
...
+        uses: actions/checkout@v2
+      - name: install deps
+        if: matrix.os == 'ubuntu-latest'
+        run: sudo apt-get update && sudo apt-get install -yq libpcsclite-dev
+      - name: build
+        run: make cosign && mv ./cosign ./cosign-${{matrix.TARGET}}-amd64
+      - name: sign
+        run: |
+          echo -n "${{secrets[matrix.COSIGN_PASSWORD]}}" | ./cosign-${{matrix.TARGET}}-amd64 sign-blob -key ./.github/workflows/cosign.key ./cosign-${{matrix.TARGET}}-amd64 > cosign-${{matrix.TARGET}}-amd64.sig
+      - name: Print Info
```
Comment (dlorenc) [inline .github/workflows/cross.yaml:41]: Does this one need to be within an "if matrix.os..." clause like the one for ubuntu on 34?

**Review by dlorenc — COMMENTED** [review]
Nice!

File: .github/workflows/cross.yaml (hunk 1 above)
Comment (pearcec) [reply .github/workflows/cross.yaml:38]: No, you want this to run for all three since they each will have different signatures.

**Review by dlorenc — APPROVED** [review]

**Review by dlorenc — APPROVED** [review]

**Comment by dlorenc** [comment]
This is awesome!

**Pull request merged**